connect/client.go

```diff
@@ -24,3 +63,25 @@ func IsRegistered() bool {
 func URLDefault() bool {
 	return CFG.BaseURL == defaultBaseURL
 }
+
+func printInformation(action string) {
+	var server string
+	if URLDefault() {
+		server = "SUSE Customer Center"
+	} else {
+		server = CFG.BaseURL
+	}
+	if action == "register" {
+		fmt.Println("Registering system to ", server)
+	} else {
+		fmt.Println("Deregistering system from ", server)
+	}
+	if CFG.FsRoot != "" {
+		fmt.Println("Rooted at: ", CFG.FsRoot)
+	}
+
+	// TODO
+	// if CFG.Email != "" {
```

Comment:
any idea where this email comes from?

Reply:
need to research that - the cli I think

Reply:
yeah, now I see that. I need to get my vision checked :o)